packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.css

```diff
@@ -4,7 +4,7 @@
 }
 
 .generated-answer__badge {
-  background-color: var(--lwc-brand-background-primary, #eef4ff);
-  color: var(--lwc-brand-primary, #1b96ff);
+  background-color: var(--lwc-colorBackgroundButtonDefaultHover, #eef4ff);
```

Comment:
Is it normal that the RGA badge is using the ButtonBackground color? 
Wouldn't it make more sense to be the brand color as it was before?

Reply:
The problem with the brand color is that it doesn't adapt to the colors set in the  community builder. that's why I had to avoid this css variable.